### ISSUE TITLE
Distributed workers sample in Java

### ISSUE THREAD

**Issue by johanandren** [akka/akka-samples#208]
Is missing, we should port the Scala sample when we have completed #186

**Comment by patriknw** [comment]
Trevor @thinkmorestupidless said that someone is working on this.

**Comment by thinkmorestupidless** [comment]
Yes, a prospect i'm working with is porting that sample to Java. I still have to confirm if they're happy to contribute the work back, though.
